cinema/models.py

```diff
@@ -130,13 +136,16 @@ def save(
     ):
         self.full_clean()
         return super(Ticket, self).save(
-            force_insert, force_update, using, update_fields
+            force_insert,
+            force_update,
+            using,
+            update_fields
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return (f"{str(self.movie_session)}"
+                f" (row: {self.row}, "
+                f" seat: {self.seat})")
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
```

Comment:
The `unique_together` option is deprecated in Django 2.2 and later. Consider using `constraints = [models.UniqueConstraint(fields=['movie_session', 'row', 'seat'], name='unique_ticket')]` instead.